### DISCUSSION TITLE
How to compile NetHook2 x64 in MVS 2022?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [SteamRE/SteamKit#1244] [General]
I created new platform, added `__x86_64__` in preprocessor definitions (because there is no `__x86_64__` in 64 mode by default in MVS) and got:
![image](https://github.com/SteamRE/SteamKit/assets/67195196/32ad813e-eeec-4e26-9180-87a3c9b5e9d5)
In x32 no problem

**Comment by yaakov-h** [comment]
Looks like a duplicate of #1243
